examples/finnegans_wake_demo/finnegans-wake-concise-demo.py

```diff
@@ -1,3 +1,19 @@
+"""
```

Comment:
Is it really necessary to have this (huge!) disclaimer even in the demo script?

I know, RMS does this, but still...

Reply:
I'm absolutely in favor of removing it.

Reply:
I like the formality in some cases.. but not this one.  Im in favor of removing it.